### PR TITLE
Made games alphabetical

### PR DESCRIPTION
**Pull request by jacobmix**
Added Crash Bandicoot 4: It's About Time, and made everything alphabetical by their game name.

### CODE REVIEW

**Comment by dafzor** [comment]
games.ini order was more based of client used. That said, Crash Bandicoot is not included on readme.

**Comment by jacobmix** [comment]
Mainly just wanted to makes it easier for the community to add games to games.ini themselves.
Easier to sort through it since it's alphabetical. I just added crash since i owned it.
Didn't add Crash 4 to the readme tho as i assume that's just a minimal example. Doesn't include all games or ptr versions.
Don't know if this changes the games.ini.example file tho. But i assume it just copies it over.

**Comment by dafzor** [comment]
The readme is actually the full "official" supported list of games.

While I've expanded the scope of bnetlauncher with epic client support it hasn't been mentioned beyond advanced configuration with the bundled examples simply being the games I play.

**Comment by jacobmix** [comment]
Alright. Added Crash Bandicoot 4 to the readme.
May be a good idea to also sort by launcher.
I'll probably edit the gamesdb.ini later tonight or tomorrow for that.

File: README.md [hunk 1]
```diff
@@ -26,22 +26,23 @@ the game starts, otherwise it will be left running.
...
 | code          | game                                                  |
 | ------------- | ----------------------------------------------------- |
-|wow            | World of Warcraft                                     |
-|wowclassic     | World of Warcraft Classic                             |
-|wowptr         | World of Warcraft Public Test Realm                   |
+|codbo4         | Call of Duty: Black Ops 4                             |
+|codbocw        | Call of Duty: Black Ops Cold War                      |
+|codmw2019      | Call of Duty: Modern Warfare (2019)                   |
+|codmw2crm      | Call of Duty: Modern Warfare 2 Campaign Remastered    |
+|wlby           | Crash Bandicoot 4: It's About Time                    |
```
Comment (dafzor) [inline README.md]: Should be cb4 not wlby

**Review by dafzor — CHANGES_REQUESTED** [review]

**Pull request merged**